.eslintrc.json

```diff
@@ -2,6 +2,42 @@
   "root": true,
   "ignorePatterns": ["**/*"],
   "plugins": ["@nx", "import", "@rnx-kit"],
+  "rules": {
+    "no-restricted-globals": [
+      "error",
+      {
+        "name": "window",
+        "message": "Get a reference to `window` from `useFluent()`."
```

Comment:
not sure if this is the right configuration file as not all packages in this repo are tightly coupled to fluent => users might get false positives with this.

Reply:
Switched to defaults for now.